website_sale_checkout_country_vat/__openerp__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+# © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Website Sale Checkout Country VAT",
+    "summary": "Autocomplete VAT in checkout process",
+    "version": "9.0.1.0.0",
+    "category": "Website",
```

Comment:
Let's go this way for now, as there's no convention on this on the rest of the modules.